src/clj_deconstruct_sigs/core.cljc

```diff
@@ -0,0 +1,148 @@
+(ns clj-deconstruct-sigs.core)
+
+#?(:clj (set! *unchecked-math* :warn-on-boxed))
```

Comment:
I think it's duplicated to `:global-vars`. I don't know whether this form is necessary.